server/NewBank.java

```diff
@@ -6,11 +6,12 @@
 public class NewBank {
 
 	private static final NewBank bank = new NewBank();
-	private HashMap<String, Customer> customers;
+	private HashMap<String, newbank.server.Customer> customers;
```

Comment:
The `newbank.server` is not necessary, and actually makes the code harder to read, please remove it